R/conversion.R

```diff
@@ -91,12 +91,9 @@ convert <- function(text,
 
   # Define Pattern for latex code
 
-  unicode_latex$int <- as.integer(as.hexmode(unicode_latex$unicode))
-  char_latex <- ifelse(unicode_latex$int <= 255 & unicode_latex$int != 177, unicode_latex$chr,
-    ifelse(unicode_latex$int < 32768,
-      paste0("\\uc1\\u", unicode_latex$int, "*"),
-      paste0("\\uc1\\u", unicode_latex$int - 65536, "*")
-    )
+  unicode_int <- as.integer(as.hexmode(unicode_latex$unicode))
+  char_latex <- ifelse(unicode_int <= 255 & unicode_int != 177, unicode_latex$chr,
+    paste0("\\uc1\\u", unicode_int - if (unicode_int < 32768) 0 else 65536, "*")
```

Comment:
Actually I think you'd better use `sprintf()` here instead of `paste0()`:

```suggestion
    sprintf("\\uc1\\u%d*", unicode_int - if (unicode_int < 32768) 0 else 65536)
```

The reason is that `paste0()` may use the scientific notation for big numbers:

```r
> sprintf('\\u%d', 1e8)
[1] "\\u100000000"
> paste0('\\u', 1e8)
[1] "\\u1e+08"
```